### DISCUSSION TITLE
Dev update Jan 8, autothrottle no longer says speed when climbing

### DISCUSSION THREAD

**Discussion by robinottawa** [flybywiresim/aircraft#2943] [General]
I got the dev update today and noticed that after taking the throttle down from TOGA to the autothrottle CL notch/detente the display reads THR CLB instead of Speed like it used to. It now changes to SPEED when I get near cruise. Looks like an improvement.

**Comment by MapleNAV-TCASFAULT** [comment]
Yes it is like that IRL
